packages/perseus-editor/src/all-editors.test.ts

```diff
@@ -1,15 +1,8 @@
-import * as React from "react";
-
 import allEditors from "./all-editors";
 
 describe("All editors", () => {
     it("should export react components with a widgetName property", () => {
         for (const editor of allEditors) {
-            expect(
```

Comment:
This was breaking after the change as `numeric-input-editor` is not a true React Class - I don't think this test was actually verifying anything useful, IMO (the next one still ensures that the `widgetName` prop exists).